### DISCUSSION TITLE
EvalAI submission keeps RUNNING for a long time

### DISCUSSION THREAD

**Discussion by Forever518** [bdd100k/bdd100k#298] [Q&A] [answered]
I have submitted my validation result to BDD100k 2D detection task on EvalAI(https://eval.ai/web/challenges/challenge-page/1857/overview) to test whether my format is right, but the status just keeps RUNNING for several days.

<img width="586" alt="1676882382420" src="https://user-images.githubusercontent.com/57933485/220054880-d34eb677-8c3e-4f3f-b6ab-0299a3b5ac37.png">

How long it will take to evaluate once? Or maybe I meet other errors like wrong format? Can someone give me a solution? Thanks!

**Comment by YWonchall** [comment]
Hi, I have the same problem, did you solve it please?

**Reply by Forever518** [reply, marked as answer]
You can directly contact EvalAI: [https://eval.ai/contact](https://eval.ai/contact). They will reply soon.

**Comment by fengxiuyaun** [comment]


**Reply by fengxiuyaun** [reply]
<img width="825" alt="image" src="https://github.com/bdd100k/bdd100k/assets/6481257/bec7a8a0-930a-465c-ae71-56ea65d4f8d4">

**Reply by fengxiuyaun** [reply]
Hi, I have the same problem, did you solve it please? After the official cancellation, I submit again and it is still running. @thomasehuang

**Reply by Forever518** [reply]
> Hi, I have the same problem, did you solve it please? After the official cancellation, I submit again and it is still running.

It may take several minutes to finish the evaluation. If it's still RUNNING after a long time, maybe the only way is to keep contacting them.

**Reply by fengxiuyaun** [reply]
> > Hi, I have the same problem, did you solve it please? After the official cancellation, I submit again and it is still running.
> 
> It may take several minutes to finish the evaluation. If it's still RUNNING after a long time, maybe the only way is to keep contacting them.
Thanks.It has been running for 5 hours。😂

**Comment by lingqwe** [comment]
Hello, I want to validate the model I trained using a test set, but I can't find how to convert the prediction results to a JSON file and upload it. Could you please share your conversion file with me

**Comment by Nimra28** [comment]
Can any one tell me the PC requirements to work on this dataset?
